kong/db/schema/entities/routes.lua

```diff
@@ -33,6 +33,7 @@ return {
                                    }, },
     { regex_priority = { type = "integer", default = 0 }, },
     { strip_path     = { type = "boolean", default = true }, },
+    { path_handling  = { type = "string", default = "v1", one_of = { "v0", "v1" }, }, },
```

Comment:
I am strongly against `v1` being the default.

Reply:
Even the community asked this many times before:
https://github.com/Kong/kong/issues/4469#issuecomment-486748567

(`v1` == `insane`, `v0` == `sane`).

Reply:
I have no strong feelings about this. I have changed the tests so they don't "assume a default" on `path_handling`. This way if we want to change this, we can just change it in the schema.

Reply:
This has been decided. V0 is the new default.